### PR TITLE
added load_from_json method to template_message and location_message for quoted_message support completion, add quoted_message handler

### PR DESCRIPTION
**Pull request by naderAbolfazli**
…from json function for message_factory

### CODE REVIEW

File: balebot/models/factories/message_factory.py [hunk 1]
```diff
@@ -34,6 +35,10 @@ def create_message(json_dict):
             return sticker_message.StickerMessage.load_from_json(json_dict)
 
         elif message_type == MessageType.json_message:
+
+            if json_message.JsonMessage.is_raw_location_message(json_dict):
+                return location_message.LocationMessage.load_from_json(json_dict)
```
Comment (baleadmin) [inline balebot/models/factories/message_factory.py]: check contacts

File: balebot/models/base_models/fat_seq_update.py [hunk 2]
```diff
@@ -20,6 +20,7 @@ def __init__(self, json):
         else:
             raise ValueError(Error.unacceptable_json)
 
+        self.json_dict = json_dict
```
Comment (baleadmin) [inline balebot/models/base_models/fat_seq_update.py]: why?

File: balebot/models/messages/location_message.py [hunk 3]
```diff
@@ -40,6 +40,7 @@ def load_from_json(cls, json):
         else:
             raise ValueError(Error.unacceptable_json)
 
+        json_dict = json_handler.loads(json_dict.get('rawJson'))
```
[the remark this answers is not in the document]
Comment (naderAbolfazli) [reply balebot/models/messages/location_message.py]: this step get through RawJsonFactory already for JsonMessage that create raw_json property of JsonMessage as a  LocationMessage or ContactMessage

File: balebot/models/factories/message_factory.py (hunk 1 above)
Comment (naderAbolfazli) [reply balebot/models/factories/message_factory.py]: done

**Pull request merged**